backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/HotSetNumberOfCpusCommand.java

```diff
@@ -88,6 +88,10 @@ protected boolean validate() {
             valid = failValidation(EngineMessage.HOT_PLUG_CPU_CONFLICT,
                     String.format("%1$s", getVm().getCpuPinningPolicy().name()));
         }
+        if (getVm().getCpuPinningPolicy() == CpuPinningPolicy.DEDICATED) {
+            valid = failValidation(EngineMessage.HOT_PLUG_CPU_IS_NOT_SUPPORTED_DEDICATED,
+                    String.format("$configuration %1$s", getVm().getCpuPinningPolicy().name()));
```

Comment:
configuration -> policy
and the change in FeatureSupported is good but it means we won't get here, right?

Reply:
Missed the second part here.
We will get here, I added a new function in FeatureSupported. Where it gets also the CPU pinning policy. We don't call it here.

Reply:
ah, there's another function there.. I thought you changed the existing one, ok